src/models/TalkCommentMapper.php

```diff
@@ -386,4 +386,40 @@ public function moderateReportedComment($decision, $comment_id, $user_id)
             }
         }
     }
+
+    /**
+     * @param integer $comment_id
+     * @return false|array
+     */
+    public function getRawComment($comment_id)
+    {
+        $sql = "select tc.* from talk_comments tc where tc.ID = :comment_id";
+        $stmt = $this->_db->prepare($sql);
+        $response = $stmt->execute(array(':comment_id' => $comment_id));
+        if ($response) {
```

Comment:
I'd recode this as follows:

```php
if (! $response) {
    return false;
}

$result = $stmt...

if (! isset($results[0]) {
    return false;
}

return $results[0];
```

That way it is more clear what the expected path of action is and where the "failures" are.

But that's merely a personal thing.